src/app/service-providers/components/ServiceProvidersTable.tsx

```diff
@@ -26,22 +36,51 @@ export type ServiceProvidersTableProps = {
 export function ServiceProvidersTable({ data }: ServiceProvidersTableProps) {
   const { isRefetching, refetch } = useProviders()
 
+  const [searchQuery, setSearchQuery] = useQueryState(
+    SEARCH_KEY,
+    parseAsString
+      .withDefault(DEFAULT_SEARCH_QUERY)
+      .withOptions({ throttleMs: 300 }),
+  )
+
+  const [sortUrlState, setSortUrlState] = useSortingQueryState()
+
+  const sortingState: SortingState = useMemo(
+    () => (sortUrlState ? [sortUrlState] : []),
+    [sortUrlState],
+  )
+
+  const handleSortingChange: OnChangeFn<SortingState> = useCallback(
+    (updater) => {
+      const newSortingState =
+        typeof updater === 'function' ? updater(sortingState) : updater
+      setSortUrlState(newSortingState[0] || null)
+    },
+    [setSortUrlState, sortingState],
+  )
+
   const table = useReactTable({
     data,
     columns,
     getCoreRowModel: getCoreRowModel(),
```

Comment:
If we aren't supporting multisort we should disable it. At the moment, it's just silenty breaking with `setSortUrlState(newSortingState[0] || null)  // Only saves first one`

```suggestion
columns,
enableMultiSort: false
getCoreRowModel: getCoreRowModel(),
```

Though, I think multi-sorting for these tables might prove useful? New ticket + GH issue?

Reply:
A breaking change with no visible error or warning...